projects/pswg/src/main/java/com/parzivail/pswg/block/ThermalDetonatorBlock.java

```diff
@@ -167,15 +169,15 @@ public void onEntityCollision(BlockState state, World world, BlockPos pos, Entit
 	@Override
 	public boolean canReplace(BlockState state, ItemPlacementContext context)
 	{
-		return !context.shouldCancelInteraction() && context.getStack().isOf(this.asItem()) && state.get(CLUSTER_SIZE) < 5 || super.canReplace(state, context);
+		return false;
 	}
 
 	@Override
 	public void onProjectileHit(World world, BlockState state, BlockHitResult hit, ProjectileEntity projectile)
 	{
 		if (projectile instanceof BlasterBoltEntity bbe)
 		{
-			Float power = world.getBlockState(hit.getBlockPos()).get(CLUSTER_SIZE) * 3f + 2;
+			Float power = world.getBlockState(hit.getBlockPos()).get(CLUSTER_SIZE) * 2f + 3;
```

Comment:
use the primitive `float` instead of the object `Float`. or, better yet, use `var` and have the type be inferred.